### DISCUSSION TITLE
Duplicate notification emails

### DISCUSSION THREAD

**Discussion by paulcripps** [solspace/craft-freeform#645] [Questions & Answers] [answered]
Hello, has anyone had issue with CRAFT 4.4.5 and Freeform 4.0.9  where admin and users get sent duplicate notification emails?

Checking and testing several times, email are complete dupes, exact same time and content.

**Comment by kjmartens** [comment]
Hi @paulcripps,

Are you using the [Dynamic Recipients](https://docs.solspace.com/craft/freeform/v4/overview/email-notifications.html#dynamic-recipients) feature here? And if so, is there any chance you have more than one option with the same email address value? 🙂 

If that's the case, then there is a current limitation where Freeform will get confused and think that the user selected both values and send multiple emails (one for each matching option).

The only workaround for this currently is to create email address aliases for each duplicate option.

**Comment by paulcripps** [comment]
Hello,

We are not using the dynamic recipients.

We are using Bootstrap 4 for our site layouts. 

Then using FreeForms Bootstrap 4 for the freeform layouts.

We are using the single form render tag.

When viewing source and looking at the output from the render tag we can TWO links of the plugin JS in the head ( /freeform/plugin.js?v= ), and in the footer with the same query string.

This is why we are getting two email sent AND we see to submissions in the FreeForm entries, both at the exact same time.

**Comment by kjmartens** [comment]
What do you have for the **Freeform Script Insertion Location** setting? (_Freeform -> Settings -> Form Behavior_)

If using `None`, please be sure to manually load Freeform's JS and CSS with the `freeform.loadFreeformPlugin()` function in your template(s). Also, be sure that you're not calling this more than once (with complex layouts, etc) and do not use this at all IF you have `Page Footer` or `Inside Form` selected. 🙂

**Comment by BridgingUnit** [comment, marked as answer]
Hi Kelsey - I'm working with Paul on this project.

Freeform Script Insertion Location setting was set to 'Page Footer', and I think it was actually behaving correctly because.... just discovered duplicate closing body/html tags in the include footer template, so Freeform carefully and conscientiously added the script to both instances! With the dupes removed the script is added the more usual single time, and it also seems to save the submission and send notifications just the once now too! Thanks for pointing us in the right direction.

Happy Easter!

**Reply by kjmartens** [reply]
Ah, that makes sense! Thanks for letting us know 🙂
